catroid/src/main/java/org/catrobat/catroid/ui/controller/LookController.java

```diff
@@ -154,7 +155,10 @@ private void handleDetails(LookData lookData, LookViewHolder holder, LookBaseAda
 				holder.lookFileSizeTextView.setText(UtilFile.getSizeAsString(new File(lookData.getAbsolutePath())));
 			}
 			int[] measure = lookData.getMeasure();
-			String measureString = measure[0] + " x " + measure[1];
+			int Length = measure[0];
+			int Width = measure[1];
```

Comment:
First! thanks for your replay 
actually this PR is no longer available, instant of 
" 84n4n4 commented on May 24 
This branch is 1 commit ahead, 89 commits behind Catrobat:develop.
this is not going to be mergable, please rebase and fix."
I deleted my repository because I didn't know how to fix that and I made a new fork. 
could you close this PR and have a look at the new PR under the Link : 
https://github.com/Catrobat/Catroid/pull/2326